test/server_metrics_test.go

```diff
@@ -0,0 +1,40 @@
+//
+// DISCLAIMER
+//
+// Copyright 2018 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+
+package test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestServerMetricsWorks tests if Client.Metrics works at all
+func TestServerMetricsWorks(t *testing.T) {
```

Comment:
```suggestion
func TestGetServerMetrics(t *testing.T) {
```

Reply:
Yes - since 3.8.0... https://www.arangodb.com/docs/devel/http/administration-and-monitoring.html#statistics-description